src/pages/myNotes/myNotesContainer.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import MyNotes from "./myNotes";
+import MyNoteDesctiption from "./myNotesDesctiption";
+import Notes from "./constants/notes";
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import PropTypes from "prop-types";
```

Comment:
Let's separate imports by their types: libraries, relative paths, absolute paths. And please set up the project with relative paths (from src directory)

```suggestion
import { useState } from "react";
import PropTypes from "prop-types";
import { makeStyles } from "@material-ui/core/styles";
import Paper from "@material-ui/core/Paper";
import Grid from "@material-ui/core/Grid";

import Notes from "src/constants/notes";

import MyNotes from "./myNotes";
import MyNoteDesctiption from "./myNotesDesctiption";
```